WordsTableTableViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  WordsTableTableViewController.swift
+//  Swift Vocabulary
+//
+//  Created by Clean Mac on 4/8/20.
+//  Copyright © 2020 LambdaStudent. All rights reserved.
+//
+
+import UIKit
+
+
+
+class WordsTableTableViewController: UITableViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        var vocabWords: [VocabularyWord] = [
+        VocabularyWord(word: "word1", definition: "Def1"),
+        VocabularyWord(word: "word2", definition: "def2"),
+        VocabularyWord(word: "word3", definition: "def3"),
+
+        ]
```

Comment:
It be a good idea to assign the 'vocabWords' outside of the viewDidLoad()

Reply:
You are missing a ending curly bracket " } " after you move the vocabWords out of the viewDidLoad() close it up with a curly bracket.